### ISSUE TITLE
Extra Start Options

### ISSUE THREAD

**Issue by Agitatio** [wasiejen/Free-Snap-Tap#4]
The one that runs without END/DEL functionality and another one that runs without a window at all would be greatly appreciated.

**Comment by wasiejen** [comment]
Hello, thank you also for trying out my program.

The start argument -nocontrols is now available. :-)
The program can then only be closed by closing the cli window.

To start without the cli window, there needs to be at least something like a system tray icon with the option to quit the program. But until now I did not get to work a system tray icon without the threating of the icon interfering with the execution of the snap tap. Tested some versions but always the same behavior and not a clue jet what happens. xD
